platform/android/SCsub

```diff
@@ -74,3 +74,10 @@ if lib_arch_dir != "":
         str(env["ANDROID_NDK_ROOT"]) + "/sources/cxx-stl/llvm-libc++/libs/" + lib_arch_dir + "/libc++_shared.so"
     )
     env_android.Command(out_dir + "/libc++_shared.so", stl_lib_path, Copy("$TARGET", "$SOURCE"))
+
+    out_openxr_dir = "#platform/android/java/lib/libs/openxr/" + lib_type_dir + "/" + lib_arch_dir
```

Comment:
Should this also be deleted in the `cleanGodotTemplates` task?

Reply:
This path is under lib/libs, which is already removed by cleanGodotTemplates.